samples/file-encryptor/enclave/encryptor.h

```diff
@@ -24,8 +22,7 @@ class ecall_dispatcher
     encryption_header_t* m_header;
 
     // initialization vector
-    unsigned char m_original_iv[IV_SIZE];
-    unsigned char m_operating_iv[IV_SIZE];
+    unsigned char m_operating_iv[SALT_SIZE_IN_BYTES];
```

Comment:
```suggestion
    unsigned char m_operating_iv[IV_SIZE];
```

Reply:
Thank you! Incorporated change suggested.